docs/SQL/SQL-Insert-Into.md

```diff
@@ -0,0 +1,106 @@
+
+**5.Handling NULL Values:**
+- The NOT operator can also be used effectively with IS NULL or IS NOT NULL to filter out or include rows with NULL values.
+
+**6.Flexibility in Subqueries:**
+- When used with subqueries, the NOT operator can help exclude sets of records efficiently.
+
+**7.Versatility in Logical Expressions:**
+- It can be used to invert the result of any logical expression, providing versatility in query construction.
+
+## Examples of Insert Into in SQL
+
+### Inserting Data into Specified Columns
+- Description :Let's assume we have a table called employees with the following columns: id, name, position, and salary.
+To insert a new employee record specifying the columns:
+- Example :
+```
+INSERT INTO employees (id, name, position, salary)
+VALUES (1, 'John Doe', 'Software Engineer', 70000);
+```
+
+
+### Inserting Data into All Columns
+- Description : If you want to insert data into all columns of the employees table, you can omit the column names:
+- Example :
+```INSERT INTO employees
+VALUES (2, 'Jane Smith', 'Data Analyst', 65000);
+```
+### Inserting Multiple Rows
+- Description :You can insert multiple rows in a single INSERT INTO statement by separating each row with a comma:
+- Example :
+
+```INSERT INTO employees (id, name, position, salary)
+VALUES 
+(3, 'Alice Johnson', 'Project Manager', 80000),
+(4, 'Bob Brown', 'UX Designer', 60000);```
+ 
+ 
+ ### Using Subqueries to Insert Data
+- Description :You can also insert data using the result of a query from another table:
+- Example :
+```INSERT INTO employees (id, name, position, salary)
+SELECT id, name, 'Intern', 30000
+FROM candidates
+WHERE status = 'Accepted';```
+
+This statement inserts data into the employees table by selecting specific columns from the candidates table where the status is 'Accepted'.
+
+### Inserting Data into Specific Columns Only
+- Description : If you don't need to insert data into all columns, you can specify only the columns you want to insert data into. The remaining columns will use their default values:
+- Example :
+```INSERT INTO employees (name, position)
+VALUES ('Charlie Davis', 'HR Specialist');```
+In this case, id and salary will take their default values (if defined).
+
+### Handling NULL Values
+- Description :If you want to insert NULL values explicitly, you can do so by specifying NULL:
+- Example :
+```INSERT INTO employees (id, name, position, salary)
+VALUES (5, 'Eve Miller', NULL, NULL);```
+## Conclusion
+The NOT operator in SQL provides a straightforward way to negate conditions in SQL queries, allowing for more flexible and precise data retrieval. Understanding its usage is crucial for crafting effective SQL statements, particularly when dealing with complex filtering requirements.
+
+---
+
+## Authors:
+
+<div style={{display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', gap: '10px'}}>
+  {['damini-chachane'].map(username => (
```

Comment:
Please replace `damini-chachane` with `Damini2004` as we need your GitHub username for fetching data from the GitHub API.